includes/Vendor/StoreListsFilter.php

```diff
@@ -63,10 +63,13 @@ public function maybe_disable_stote_lists_filter() {
      * @return void
      */
     public function filter_area( $stores ) {
+        $sort_by = dokan_get_option( 'store_list_sort_by','dokan_appearance', 'most_recent' );
```

Comment:
Please add a space between the first 2 arguments here in line 66. Also, add a space between the first 2 arguments in line 104.